### DISCUSSION TITLE
How to create the nice cover image

### DISCUSSION THREAD

**Discussion by Alaye-Dong** [kungfux/kungfux.github.io#8] [Q&A] [answered]
> ![image](https://github.com/user-attachments/assets/7fb255d3-7d59-4c37-9991-0a143c4b04e1)

Hello😄, I visited the blog website and found the cover image to be quite simple yet beautiful, featuring a theme color with the corresponding icon positioned in the upper right corner. 🤩

🤔It seems as though it was made using a template.  I'm curious to learn how it was made. Please! 🫡

**Comment by kungfux** [comment, marked as answer]
Hey @Alaye-Dong 

Thanks 😊 

Your guess is correct. I have created a template that I use to create a cover for each of my posts. I use [Figma](figma.com) for this purpose. It allows me to create a stylesheet with colors and align the text and icon properly.
